src/Themes/NexusMods.Themes.NexusFluentDark/Styles/Controls/Button/PillStyles.xaml

```diff
@@ -219,12 +219,8 @@
 
     <!-- Pill Label -->
     <!-- Labels should have all interactions disabled, so they don't look interactable -->
+    <!-- NOTE(insomnious): left this empty selector here to make it easier to find in future -->
     <Style Selector="Button.Pill.Label">
-
-        <!--  Disable the scaling found in Fluent buttons when pressing -->
-        <Style Selector="^:pressed">
```

Comment:
This should probably stay.
The pills, like `Premium` should be non-interactable generally. 
By removing the 1.0, Avalonia applies the default, which is to shrink.